Imgsmall.java

```diff
@@ -0,0 +1,34 @@
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.awt.image.RenderedImage;
+import java.io.File;
+import java.net.URL;
+
+public class Imgsmall {
+  public static void main(String[] args) throws Exception{
```

Comment:
Instead of writing your entire logic in main method, write it in a proper method like resize() or reduceSize() something like that and use main() method to drive the program (meaning test it)

Reply:
Do not throw a generic exception from your methods, never from main() anyway.